app/views/TrusteesInfoView.scala.html

```diff
@@ -76,5 +76,5 @@ <h2>@messages("trusteesInfo.subheading2")</h2>
         </strong>
     </div>
 
-    @components.button_link(messages("trusteesInfo.continue"), TrusteeIndividualOrBusinessController.onPageLoad(NormalMode, 0).url)
+    @components.button_link(messages("site.justContinue"), TrusteeIndividualOrBusinessController.onPageLoad(NormalMode, 0).url)
```

Comment:
This should be navigating them to IsThisTheLeadTrustee only if they have no trustees added.

Reply:
Done